Update Fast fourier transform ahora si.cpp

oscarsierra1 opened the pull request.

oscarsierra1 committed:
Update Fast fourier transform ahora si.cpp

osdajigu commented on Math/Fast fourier transform.cpp:
```diff
@@ -1,48 +1,50 @@
+#define rep(i, a, b) for(int i = a; i < (b); ++i)
+#define sz(v) ((int)v.size())
+#define trav(a, x) for(auto& a : x)
+#define all(v) v.begin(),v.end()
+ 
+typedef vector<ll> vl;
+typedef vector<int> vi;
+ 
+ 
```
```suggestion
#define all(v) v.begin(),v.end()
typedef vector<ll> vl;
typedef vector<int> vi;
```

osdajigu reviewed (commented):
Please leave this 
![image](https://github.com/ahoraSoyPeor/notebook_descomUNAL/assets/13740279/3c789bab-be08-4b0c-9a51-8198a9d9b7fb)
and add the new link to the problem

oscarsierra1 committed:
Update Fast fourier transform.cpp

previous version of fft gives WA test 24 in this problem  also it runs in 826 ms
new version of fft gives AC in the same problem and runs in 390 ms
